multiple-event-types-protobuf/kafka/src/main/proto/customer-event.proto

```diff
@@ -3,15 +3,15 @@ syntax = "proto3";
 package io.confluent.developer.proto;
 
 import "purchase.proto";
-import "page-view.proto";
+import "pageview.proto";
 
 option java_outer_classname = "CustomerEventProto";
```

Comment:
For all the protobuf files I'd add `option java_multiple_files = true` that way you can reference them in the Java code just by the class name and not the awkward Outerclass.ClassName.  

I wasn't aware of that when I did this originally

Reply:
done!